WordPress/gutenberg
# Update overlay step function in cover block

## What?

Fixes https://github.com/WordPress/gutenberg/issues/51625

## Why?

Currently, we can set overlay in multiple of 10 in cover block.

## How?

I have updated steps to enable any number from 1 to 100 to be set as overlay value.

## Testing Instructions




1. Add cover block
2. Change overlay
3. Notice opacity
 
## Screenshots or screencast 
![CleanShot 2024-03-15 at 12 22 15@2x](https://github.com/WordPress/gutenberg/assets/61308756/6c9311aa-f79d-4966-bb46-b4024099e245)
